lib/bugsnag/exception_notification.rb

```diff
@@ -33,5 +36,45 @@ def call(exception, options={}, &block)
 
       Bugsnag.notify(exception, &wrapped_block)
     end
+
+    private
+
+    # `:env` option can be passed as a Rack Env object by `ExceptionNotification::Rack`.
+    # Bugsnag provides the way to report the Env object by default.
+    #
+    # @see https://github.com/smartinez87/exception_notification/blob/v4.3.0/lib/exception_notification/rack.rb#L51
+    # @see https://github.com/bugsnag/bugsnag-ruby/blob/v6.0.0/lib/bugsnag/middleware/rack_request.rb#L61-L63
+    # @see https://docs.bugsnag.com/platforms/ruby/rails/configuration-options/#send_environment
+    def support_rack?(key)
+      key == :env
+    end
+
+    def process_rack(report, value)
+      if report.configuration.send_environment
+        report.add_tab(:environment, value)
+      end
+    end
+
+    # Support Sidekiq data format.
+    #
+    # @see https://github.com/smartinez87/exception_notification/blob/v4.3.0/lib/exception_notification/sidekiq.rb#L28
+    def support_sidekiq?(key, value)
+      key == :data && value.is_a?(Hash) && value.include?(:sidekiq)
+    end
+
+    def process_sidekiq(report, value)
+      report.add_tab(:sidekiq, value[:sidekiq])
```

Comment:
`bugsnag-ruby` calls not only `add_tabl` but also `context=`. But I think calling `context=` also depends too much on the internal implementation of `bugsnag-ruby` and it's very hard to maintain `exception_notification-bugsnag`.
So, I omitted `context=`. What do you think?

https://github.com/bugsnag/bugsnag-ruby/blob/v6.11.1/lib/bugsnag/middleware/sidekiq.rb#L12-L13